### PR TITLE
refactor(useValueRef): import useValueRef from maskbook-shared directly

### PR DESCRIPTION
**Pull request by UncleBill**
closes #

### CODE REVIEW

**Review by Jack-Works — APPROVED** [review]

**Comment by zhouhanseng** [comment]
Hi, could you please merge this later after https://github.com/DimensionDev/Maskbook/pull/2374 get merged? It tooks me 1-2 hours to solve rebase conflict each time. Thanks!

**Comment by UncleBill** [comment]
> Hi, could you please merge this later after https://github.com/DimensionDev/Maskbook/pull/2374 get merged? It tooks me 1-2 hours to solve rebase conflict each time. Thanks!

No problem.

**Review by Jack-Works — APPROVED** [review]

**Review by Jack-Works — DISMISSED** [review]

**Review by guanbinrui — DISMISSED** [review]

**Review by Jack-Works — DISMISSED** [review]

**Comment by UncleBill** [comment]
I merge and build locally, it's been a long day since the last rebasing, but the building still works after merging.

Merging now.

**Pull request merged**
